packages/@react-spectrum/layout/src/Grid.tsx

```diff
@@ -45,7 +45,7 @@ function Grid(props: GridProps, ref: DOMRef<HTMLDivElement>) {
     children,
     ...otherProps
   } = props;
-  let {styleProps} = useStyleProps(otherProps, gridStyleProps);
+  let {styleProps} = useStyleProps(otherProps, {handlers: gridStyleProps});
```

Comment:
i think useStyleProps is only internal so we should be fine, but just making sure it's brought up to be checked off